test/integration.sh

```diff
@@ -47,12 +33,10 @@ echo "$phoned/$minPhoned machines have phoned home"
 sleep 10
 
 echo "Adding route to leaf01"
-sudo ip r d 100.255.254.0/24 || true
-$(make route) || true
+make route
```

Comment:
This just echos the command to add the route to stdout but the command does not get executed because there is no subshell opened!

Reply:
Ah!